src/pages/TermsOfService.ts

```diff
@@ -0,0 +1,149 @@
+import _ from 'lodash/fp';
+import { useState } from 'react';
+import { div, h, h1, img } from 'react-hyperscript-helpers';
+import { ButtonOutline, ButtonPrimary, ButtonSecondary } from 'src/components/common';
+import { centeredSpinner } from 'src/components/icons';
+import { MarkdownViewer, newWindowLinkRenderer } from 'src/components/markdown';
+import scienceBackground from 'src/images/science-background.jpg';
+import { Ajax } from 'src/libs/ajax';
+import { signOut } from 'src/libs/auth';
+import colors from 'src/libs/colors';
+import { reportError, withErrorReporting } from 'src/libs/error';
+import * as Nav from 'src/libs/nav';
+import { useOnMount, useStore } from 'src/libs/react-utils';
+import { authStore } from 'src/libs/state';
+import * as Style from 'src/libs/style';
+import * as Utils from 'src/libs/utils';
+
+const TermsOfServicePage = () => {
+  const [busy, setBusy] = useState<boolean>();
+  const { signInStatus, termsOfService } = useStore(authStore);
+  const acceptedLatestTos = signInStatus === 'signedIn' && termsOfService.userHasAcceptedLatestTos;
+  const usageAllowed = signInStatus === 'signedIn' && termsOfService.permitsSystemUsage;
+  const [tosText, setTosText] = useState();
```

Comment:
This state could use a type.

```suggestion
  const [tosText, setTosText] = useState<string>();
```